### PR TITLE
MCS-1425 Novel containers, obstacles, occluders and passive physics objects.

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
This PR is just new/modified prefabs and corresponding updates to the "object registry" files.

YOU WILL ALSO NEED THE NEW UNITYASSETSTORE PREFABS: https://github.com/NextCenturyCorporation/mcs-private/pull/390

Test scene: eval_6_novel

### CODE REVIEW

**Review by phile-caci — APPROVED** [review]
Looks good, rebuilt addressables, tested scenes with new assets.

File: unity/Assets/AddressableAssetsData/AddressableAssetSettings.asset [hunk 1]
```diff
@@ -163,7 +163,7 @@ MonoBehaviour:
   - {fileID: 11400000, guid: f90d2048f1cf38745afedd32f3bcb792, type: 2}
   - {fileID: 11400000, guid: ec99645f55e1efe43ac3838dc50554b9, type: 2}
   - {fileID: 11400000, guid: b0d524d2f6e6ee34da2e410e714bed0f, type: 2}
-  m_ActiveProfileId: ed5e466c69b02154d8732475103c81cf
+  m_ActiveProfileId: ca1c2555f02a867488855f32722df6bc
```
Comment (rartiss55) [inline unity/Assets/AddressableAssetsData/AddressableAssetSettings.asset:166]: Did you mean to change this from localhost?

**Review by rartiss55 — APPROVED** [review]
Once I remembered to recheck my profile, got everything working + can see all the new objects.

**Review by jaudick — APPROVED** [review]

**Review by brianpippin — APPROVED** [review]

**Comment by ThomasSchellenbergNextCentury** [comment]
I ran scenes from python containing novel material tools, novel width tools, and other novel objects, and they all worked, so I'm pretty sure I merged everything correctly.

**Pull request merged**